server/src/main/java/io/druid/server/emitter/ParametrizedUriEmitterModule.java

```diff
@@ -63,6 +64,7 @@ public Emitter getEmitter(
     if (sslContext != null) {
```

Comment:
Could this common logic be shared between in HttpEmitterModule and ParametrizedUriEmitterModule?

Reply:
Yeah, I think so. But I am having a hard time to come up with a proper place to put the common code. Previously, part of the logic was in `java-util`'s `HttpClientInit.createClient`; maybe we should have a similar factory method for `AsyncHttpClient` in `java-util`?

Reply:
it may be a static method in HttpEmitterModule, and ParametrizedUriEmitterModule uses it

Reply:
You're right, I missed the other static method in `HttpEmitterModule`. Thanks :+1: